README.md

```diff
@@ -33,6 +33,19 @@ Application Options:
   --sign-args                 Extra arguments to use when signing with gpg[2]
 ```
 
+
+### Environment variables
+
+Some of the parameters above can alternatively be specified as environment variables. Also, some additional environment variables are available, too.
+
+- `ARCH`: Needs to be set whenever appimagetool cannot automatically determine the architecture of the binaries inside the AppDir to choose a suitable runtime (e.g., when binaries for multiple architectures or just shell scripts are contained in there).
+- `APPIMAGETOOL_FORCE_SIGN`: By default, if signing fails, appimagetool just logs a warning but will not abort immediately. If this environment variable is set, appimagetool exits with a non-zero return code.
```

Comment:
Do we need this?

Reply:
Well, I'd rather get rid of it and make "signing fails" an error case by default. *Nobody* should ever run in a case where signing fails and appimagetool should just continue. You can always remove the sign flag from the CLI for such cases...

Reply:
That is for another issue, though. If we decide to make this change, we can do it now and rebase this PR later on.

Reply:
Agree, if `-s` was requested then failures should be fatal.

Reply:
https://github.com/AppImage/appimagetool/issues/19